### PR TITLE
refs #8635 - empty hammer-cli-gutterball module

### PR DESCRIPTION
**Pull request by komidore64**

### CODE REVIEW

**Comment by komidore64** [comment]
oops, need to enable travis-ci

File: rel-eng/tito.props [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[buildconfig]
+builder = tito.builder.GemBuilder
+tagger = tito.tagger.VersionTagger
+changelog_do_not_remove_cherrypick = 0
+changelog_format = %s (%ae)
```
Comment (mccun934) [inline rel-eng/tito.props:5]: Probably need the right koji tags here to:

[katello-nightly-rhel6]
disttag = .el6
blacklist =

[katello-nightly-rhel7]
disttag = .el7
blacklist =

Comment (komidore64) [inline rel-eng/tito.props:5]: roger that

Comment (komidore64) [inline rel-eng/tito.props:5]: addressed.

File: rubygem-hammer_cli_gutterball.spec [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+%dir %{geminstdir}
+%{geminstdir}/
+%config(noreplace) %{_sysconfdir}/%{confdir}/cli.modules.d/gutterball.yml
+%exclude %{gem_dir}/cache/%{gemname}-%{version}.gem
+%{gem_dir}/specifications/%{gemname}-%{version}.gemspec
+
+%files doc
+%doc %{gem_dir}/doc/%{gemname}-%{version}
+
+%changelog
```
Comment (mccun934) [inline rubygem-hammer_cli_gutterball.spec:68]: uh .. giving me credit here? shouldn't this be you?

Comment (komidore64) [inline rubygem-hammer_cli_gutterball.spec:68]: oh haha. i copypasta'd hammer-cli-sam's spec

Comment (komidore64) [inline rubygem-hammer_cli_gutterball.spec:68]: removed

**Comment by mccun934** [comment]
ACk

**Pull request merged**